test/sql/parallelism/interquery/test_concurrentupdate.cpp

```diff
@@ -20,6 +20,11 @@ TEST_CASE("Single thread update", "[interquery][.]") {
 	DuckDB db(nullptr);
 	Connection con(db);
 
+    // enable detailed profiling
+    con.Query("PRAGMA enable_profiling");
+    con.Query("PRAGMA profiling_output='__TEST_DIR__/test.json'");
```

Comment:
__TEST_DIR__ like this will only work in the sqllogictests. Do you need to write the profiling output to a file? If so, use TestCreatePath from test_helpers.hpp.

Reply:
I think if the profiling output doesn’t go into a file, it would spam the testing output with the result of each query.